hooks/base16-delta.sh

```diff
@@ -0,0 +1,104 @@
+#!/usr/bin/env bash
+
+# ----------------------------------------------------------------------
+# Setup config variables and env
+# ----------------------------------------------------------------------
+
+DELTA_GITCONFIG_PATH="$BASE16_CONFIG_PATH/delta.gitconfig"
+
+if [ -f "$DELTA_GITCONFIG_PATH" ]; then
```

Comment:
```suggestion
if [ ! -e "$DELTA_GITCONFIG_PATH" ]; then
```

Since this creates a file with `touch`, I'm guessing the above test should be conditional of the file not existing `! -f`. I suggest using `-e` instead of `-f` since it's possible the delta.gitconfig file is a symlink, which `-e` catches and `-f` doesn't.

Reply:
Good catch! These lines in `base16-vim.sh` must also be bugs then: https://github.com/tinted-theming/base16-shell/pull/31